### ISSUE TITLE
Repo: Playground linting and tab switching don't trigger locally on specifically JoshuaKGoldberg's Macs

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [typescript-eslint/typescript-eslint#6368]
### Suggestion

I can't get the playground to work locally 😞. Clicking the _tsconfig_ and _eslintrc_ tabs does nothing. 

<img width="691" alt="Screenshot of the playground with 'visual editor' visible on top of the code editor" src="https://user-images.githubusercontent.com/3335181/214042161-2345b7c9-7b78-41d1-ba7c-de092f642220.png">

Note that this happens even if I revert #5663 locally (`git revert fab9974`).

**Comment by armano2** [comment]
this is surprising, as this is working fine for me

**Comment by JoshuaKGoldberg** [comment]
Hmm, I tried on both macOS and Windows with Chrome and Firefox each. Windows works but macOS does not. 🤔

**Comment by Josh-Cena** [comment]
Can't reproduce on macOS Chrome/Firefox/Safari

**Comment by JoshuaKGoldberg** [comment]
[🫠](https://emojipedia.org/melting-face/)

**Comment by JoshuaKGoldberg** [comment]
Tried it again today and got this stack trace:

```plaintext
TypeError: Cannot read properties of undefined (reading 'convertCompilerOptionsFromJson') at createCompilerOptions (webpack-internal:///./src/components/lib/createCompilerOptions.ts:7:71) at eval (webpack-internal:///./src/components/editor/useSandboxServices.ts:16:520)
```

The line numbers are off but seem to come from:

https://github.com/typescript-eslint/typescript-eslint/blob/a5f10ff244b2d4f33b21617d631e5fc40fb24c2e/packages/website/src/components/lib/createCompilerOptions.ts#L9

Executing `window.ts` in the dev tools console shows that it indeed does not exist.

**Comment by armano2** [comment]
can you try running it in incognito? as if ts is not being loaded, we may be dealing with CORS issue or one of your addon in browser is acting out.

**Comment by JoshuaKGoldberg** [comment]
The issue went away this morning on a fresh install! I don't know what the change was (it hasn't been the first time I've switched computers even) but 🤷 hey I'll take it!
